app/controllers/api/v1/gogovan_orders_controller.rb

```diff
@@ -23,7 +23,11 @@ class GogovanOrdersController < Api::V1::ApiController
       param :offerId, String, desc: "Id of the offer"
       def calculate_price
         order_price = GogovanOrder.place_order(current_user, order_params)
-        render json: order_price.to_json
+        if order_price[:error]
+          render_error(order_price[:error])
```

Comment:
What kind of error does it throw? Is it full_message or with status code? (As per my knowledge it will through error including status code like an example of 503 with service not available,)🤔 

https://github.com/crossroads/go_go_van_api/blob/master/lib/go_go_van_api/order.rb#L38

Looks like it sends with status code so I am not sure we should show that way to a user or not. Let me know your thoughts on this. :)

Could you please add a screenshot of error how it displays on user side?